Doc fix

cvvergara opened the pull request:
- [x] fix some communication issues with transifex
- [x] fix resul queries of lineGraph
- [x] Updates the documentation strings for translating

@pgRouting/admins

vidhan13j07 commented:
@cvvergara It seems the queries in `doc-pgr_lineGraph.queries` are changed. [These](https://github.com/pgRouting/pgrouting/blob/develop/doc/queries/doc-pgr_lineGraph.queries) queries are to be used.

cvvergara commented:
@vidhan13j07 
The queries file are automatically generated using:
tools/testers/algorithm-tester.pl -alg lineGraph -documentation
I just ran the command and committed the change.
Because tests pass the file should look exactly as `src/lineGraph/test/doc-pgr_lineGraph.result``
```
diff src/lineGraph/test/doc-pgr_lineGraph.result doc/queries/doc-pgr_lineGraph.queries 
```
btw, I deleted the "non approved change" in sampledata.sql that was making an edge_table255
I also deleted the non used restrict table you made, because at the end pgr_dijkstraTRSP cound not be finished, so its not used.

"non aproved change" is for the release, it is approved for your GSoC work on your branch
